src/components/MultiSelect/styles.scss

```diff
@@ -64,6 +64,16 @@
     align-items: flex-start;
     justify-content: space-between;
 
+    &:hover {
+      border: $tw-sz-frm-fld-text-input-stroke solid $tw-clr-brdr-hover;
+    }
+
+    &.open {
+      border: $tw-sz-frm-fld-text-input-stroke solid $tw-clr-brdr-active;
```

Comment:
Nice! I haven't seen or used any `tw-sz` properties yet, nice to know even the little things are spec'd out